middle_end/flambda/types/env/typing_env_extension.rec.mli

```diff
@@ -21,6 +21,10 @@
 
 type t
 
+include Expr_std.S with type t := t
```

Comment:
Is there a reason to have this ?

Reply:
free_names is required for Row_like.free_names

Reply:
You have `Contains_names.S` for this (that's what most of the other signatures in the `types` folder use)